test/functional/rawtransactions.py

```diff
@@ -15,6 +15,16 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import *
 
+
+class multidict(dict):
```

Comment:
Can you add a comment for this please? Something like:

```
"""Dictionary that allows duplicate keys.

Constructed with a list of (key, value) tuples. When dumped by the json module, will output invalid json with repeated keys, eg:
>>> json.dumps(multidict([(1,2),(1,2)])
'{"1": 2, "1": 2}'

Used to test calls to rpc methods with invalid repeated keys in the json object."""
```

Reply:
Yeah submitting JSON objects in deterministic order and with potentially duplicated keys is the only thing it's good for, it's not quite a traditional multidict implementation.

Reply:
Done.